frontend/components/common/formatters/AnimatedNumber.vue

```diff
@@ -32,8 +33,18 @@ export default Vue.extend({
             type: Number,
             default: undefined,
         },
+        formatted: {
+            type: Boolean,
+            default: false,
+        },
     },
     computed: {
+        safeValue(): number {
+            if (BigNumber.isBigNumber(this.value)) {
+                return this.value.toNumber();
+            }
+            return this.value;
+        },
```

Comment:
Why is this needed? We should not covert big numbers right before we want to display them because this might produce:
- big numbers being displayed using scientific notation
- small numbers loosing precision

Reply:
We did this beforehand as well. Beforehand we passed the `format(value)` as the value to the animated Number component. The external `animated-number` component does not support BigNumber, so we need to format it to a number before we can pass it. 

Before this PR, we passed it with our format function, which outputted a string, that could still be parsed as a int. With the new function, a string is returned that cannot be formatted back into a number, resulting in an error. 

This is why I added a `safeValue`, which is always a number. This number is the base number, the animated-number component uses. Then I pass the format function to it, which it calls and then the decimal place is added and the string is correctly formatted.

Reply:
> so we need to format it to a number before we can pass it

Not true. The formatting function as well as the `value` prop [accepts string or number](https://www.npmjs.com/package/animated-number-vue#props). Therefore it's preferable to send string.

> This is why I added a `safeValue`

As discussed many times in this repo, we can not use `.toNumber()` under any circumstances since we risk loosing precision and/or formatting a number with scientific notation, which most people wouldn't understand